assignments/prototypes.js

```diff
@@ -39,6 +39,33 @@
   * Instances of CharacterStats should have all of the same properties as GameObject.
 */
 
+function GameObject(gameObejectAttributes){
+  this.createdAt = gameObejectAttributes.createdAt;
+  this.dimensions = gameObejectAttributes.dimensions;
+}
+
+GameObject.prototype.destroy = function () {
+  return `${this.name} was removed from the game`; 
+}
+
+function CharacterStats(characterStatsAttribute){
+  this.hp = characterStatsAttribute.hp;
+  this.name = characterStatsAttribute.name; 
+  GameObject.call(this, characterStatsAttribute); 
+}
+
+CharacterStats.prototype = Object.create(GameObject.prototype); 
+
+CharacterStats.prototype.takeDamage = function () {
+  return `${this.name} took damage`; 
+} 
+
+function Humanoid(humanoidAttributes){  
+  this.faction = humanoidAttributes.faction; 
+  this.weapons = humanoidAttributes.weapons; 
+  this.language = humanoidAttributes.language; 
+  CharacterStats.call(this,humanoidAttributes);
+}
```

Comment:
```suggestion
}
Humanoid.prototype = Object.create(CharacterStats.prototype); 

Humanoid.prototype.greet = function () {
  return `${this.name} extends a greeting in ${this.language}`; 
} 
```
Gotta connect the methods, and add the greet method